### ISSUE TITLE
Ploting traces as function of time (not frames) 

### ISSUE THREAD

**Issue by mcarbotano** [MouseLand/suite2p#145]
Hi again!
First of all, many thanks to you two, this little monster is getting better and better!

Now, I would like to know if it possible to introduce a option to plot the results not as frames but as a function of time. I have my behavior data collected at around 250Hz and the imaging at  8Hz (tops). I'm trying to reduce the 75.000 behavior frames to 1000 (2p) to load it to the GUI but its of no use. Im loosing many of the behavior events (they last less than seconds).
If the GUI could handle time stamps instead of frames could it be done? Also, the GUI should be able to handle differences in point numbers.

Thanks a ton!

**Comment by carsen-stringer** [comment]
I think handling timestamps would be pretty straightforward (just load a numpy with 2 columns, one with timestamps) - but I won't be able to get to this until the weekend, sorry

what do you mean by differences in point numbers?

**Comment by mcarbotano** [comment]
Thanks a lot!
By differences in point numbers I meant the possibility of adding around 60.000 behaviors frames to 800 of imaging.
I read Marius in some issue saying that the time point must be same. He suggested down sampling the the behavior data. Probably I'm getting it wrong.

**Comment by carsen-stringer** [comment]
I've added this to the github version - load an *.npy file with two columns (NT x 2) where the second column is the timestamps

**Comment by mcarbotano** [comment]
Thank a ton!
I'll try it ASAP

**Comment by mcarbotano** [comment]
Hi!
We tried to load behavior data in the github version and nothing happens. I tried many different .npy arrays shapes, 1D, 2D, long and wide format, and I get nothing showed.

In the console we get:
`*Traceback (most recent call last):
  File "/home/mathilde.lapoix/anaconda3/lib/python3.7/site-packages/suite2p/gui2p.py", line 1253, in load_behavior
    fig.beh_masks(self)
  File "/home/mathilde.lapoix/anaconda3/lib/python3.7/site-packages/suite2p/fig.py", line 394, in beh_masks
    nb = int(np.floor(parent.beh.size/parent.bin))
ZeroDivisionError: division by zero
8`

Its strange but we  were not able to load a 1D behavior trace to the pip version (we even tried using the same frames number as the imaging).
